xbmc/windowing/WinSystem.h

```diff
@@ -120,6 +120,19 @@ class CWinSystemBase
   virtual bool IsTextInputEnabled() { return false; }
 
   std::string GetClipboardText(void);
+  
+  /**
+   * Get VADisplay that can be used for hardware decoding
+   * 
+   * Kodi currently does not use libva backend-specific interfaces for presenting
+   * the decoded frames.
+   * If this changes, the API for interacting with the windowing system
+   * will have to be augmented.
+   * 
+   * \return new VADisplay that can be used for hardware decoding or nullptr
+   *         if unsupported or creation failed
+   */
+  virtual void* GetVaDisplay() { return nullptr; }
```

Comment:
I think I'll just drop the virtual and do the big cleanup when I start #11. It will be a really large refactor anyway and this line will be the smallest of my problems :-)